app/views/storage_locations/_storage_location_row.html.erb

```diff
@@ -2,6 +2,7 @@
   <td><%= storage_location.name %></td>
   <td><%= storage_location.address %></td>
   <td><%= storage_location.size %></td>
+  <td><%= storage_location.total_inventory_fair_market_value %>
```

Comment:
You'll want to pass this to `number_to_currency` 
 ( https://apidock.com/rails/ActionView/Helpers/NumberHelper/number_to_currency ) -- currently it is just returning the amount in cents. (100 cents in $1)

Reply:
We have a helper we use that does this in `ItemsHelper` that you might want to take a look at.  Look in `/app/helpers/items_helper.rb`